ipmi/warewulf-ipmi.spec.in

```diff
@@ -1,46 +1,43 @@
 %{!?_rel:%{expand:%%global _rel 0.%(test "@GITVERSION@" != "0000" && echo "@GITVERSION@" || git show -s --pretty=format:%h || echo 0000)}}
-
 %define debug_package %{nil}
+%{!?wwsrvdir:%{expand:%%define wwsrvdir %{_localstatedir}/warewulf}}
 
-Summary: IPMI Module for Warewulf
+Summary: Warewulf - IPMI support
 Name: warewulf-ipmi
 Version: @PACKAGE_VERSION@
 Release: %{_rel}%{?dist}
-#Release: 1%{?dist}
 License: US Dept. of Energy (BSD-like)
-Group: System Environment/Clustering
 URL: http://warewulf.lbl.gov/
 Source: %{name}-%{version}.tar.gz
 ExclusiveOS: linux
 Conflicts: warewulf < 3
-Requires: warewulf-common %{name}-initramfs-%{_arch} = %{version}-%{release}
+Requires: warewulf-common
+Requires: %{name}-initramfs-%{_arch} = %{version}-%{release}
 
 %if 0%{?rhel} >= 8 || 0%{?sle_version} >= 150000
+%global localipmi 1
 BuildRequires: ipmitool
 Requires: ipmitool
 %define CONF_FLAGS "--with-local-ipmitool=yes"
+%else
+%global localipmi 0
+Provides: ipmitool = 1.8.18
```

Comment:
Do we actually want to advertise that this package provides ipmitool for the system?